routes/index.js

```diff
@@ -20,6 +20,7 @@ routes.GETemailver = userRoutes.GETemailver;
 routes.GETallUsers = userRoutes.GETallUsers;
 routes.POSTchangeAdmin = userRoutes.POSTchangeAdmin;
 routes.POSTchangePassword = userRoutes.POSTchangePassword;
+routes.POSTdelUser = userRoutes.POSTdelUser;
```

Comment:
You can actually make HTTP `DELETE` requests. Maybe this would be proper here? i have no idea